src/app/components/Modals/WithdrawV2Modal.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  Button,
+  Dialog,
+  DialogBody,
+  DialogFooter,
+  DialogHeader,
+  Text,
+} from "@babylonlabs-io/bbn-core-ui";
+
+import { getNetworkConfig } from "@/config/network.config";
+interface UnbondModalProps {
+  open: boolean;
+  onClose: () => void;
+  onProceed: () => void;
+  awaitingWalletResponse: boolean;
+}
+
+const { networkName } = getNetworkConfig();
+
+export const WithdrawModalV2 = ({
```

Comment:
The name of the component is `WithdrawModalV2`
The name of the file is `WithdrawV2Modal`
I suggest we use one of two for both cases